src/data/real/nnreal.lean

```diff
@@ -642,6 +642,12 @@ lemma half_lt_self {a : ℝ≥0} (h : a ≠ 0) : a / 2 < a :=
 by rw [← nnreal.coe_lt_coe, nnreal.coe_div]; exact
 half_lt_self (bot_lt_iff_ne_bot.2 h)
 
+lemma lt_mul_two_self {a : ℝ≥0} (ha : a ≠ 0) : a < 2 * a :=
```

Comment:
Is there a more general version that works in some class of ordered semirings instead of just nnreal? If so, we should rather have the general version. Also, the name should rather be `lt_two_mul_self`.

Reply:
And you can prove it by applying `lt_mul_of_one_lt_left`, which works in an ordered semiring.